src/taoensso/faraday.clj

```diff
@@ -369,15 +411,18 @@
 
   TableDescription
   (as-map [d]
-    {:name          (keyword (.getTableName d))
-     :creation-date (.getCreationDateTime d)
-     :item-count    (.getItemCount d)
-     :size          (.getTableSizeBytes d)
-     :throughput    (as-map (.getProvisionedThroughput  d))
-     :indexes       (as-map (.getLocalSecondaryIndexes  d)) ; DEPRECATED
-     :lsindexes     (as-map (.getLocalSecondaryIndexes  d))
-     :gsindexes     (as-map (.getGlobalSecondaryIndexes d))
-     :status        (utils/un-enum (.getTableStatus d))
+    {:name                 (keyword (.getTableName d))
+     :creation-date        (.getCreationDateTime d)
+     :item-count           (.getItemCount d)
+     :size                 (.getTableSizeBytes d)
+     :throughput           (as-map (.getProvisionedThroughput d))
+     :indexes              (as-map (.getLocalSecondaryIndexes d)) ; DEPRECATED
+     :lsindexes            (as-map (.getLocalSecondaryIndexes d))
+     :gsindexes            (as-map (.getGlobalSecondaryIndexes d))
+     :stream-specification (as-map (.getStreamSpecification d))
```

Comment:
Perhaps `:stream-spec` would be more convenient + sufficiently unambiguous?

To clarify: suggesting a change of this and all other API uses of `:stream-specification`.